diff --git a/coldfront/core/allocation/management/commands/add_allocation_defaults.py b/coldfront/core/allocation/management/commands/add_allocation_defaults.py
index 9f7dc01..3fd2fa7 100644
--- a/coldfront/core/allocation/management/commands/add_allocation_defaults.py
+++ b/coldfront/core/allocation/management/commands/add_allocation_defaults.py
@@ -22,6 +22,7 @@ class Command(BaseCommand):

         for choice in (
             "Active",
+            "Active (Needs Renewal)",
             "Denied",
             "Expired",
             "New",
diff --git a/coldfront/core/allocation/tasks.py b/coldfront/core/allocation/tasks.py
index 883420a..3247ce3 100644
--- a/coldfront/core/allocation/tasks.py
+++ b/coldfront/core/allocation/tasks.py
@@ -36,7 +36,7 @@ EMAIL_ALLOCATION_EULA_IGNORE_OPT_OUT = import_from_settings("EMAIL_ALLOCATION_EU


 def update_statuses():
-    expired_status_choice = AllocationStatusChoice.objects.get(name="Expired")
+    expired_status_choice = AllocationStatusChoice.objects.get(name="Active (Needs Renewal)")
     allocations_to_expire = Allocation.objects.filter(
         status__name__in=[
             "Active",
