diff --git a/coldfront/core/allocation/management/commands/add_allocation_defaults.py b/coldfront/core/allocation/management/commands/add_allocation_defaults.py
index ca5c4fc..556eeb2 100644
--- a/coldfront/core/allocation/management/commands/add_allocation_defaults.py
+++ b/coldfront/core/allocation/management/commands/add_allocation_defaults.py
@@ -16,7 +16,7 @@ class Command(BaseCommand):
             'Attribute Expanded Text'):
             AttributeType.objects.get_or_create(name=attribute_type)
 
-        for choice in ('Active', 'Denied', 'Expired',
+        for choice in ('Active (Needs Renewal)', 'Active', 'Denied', 'Expired',
                        'New', 'Paid', 'Payment Pending',
                        'Payment Requested', 'Payment Declined',
                        'Renewal Requested', 'Revoked', 'Unpaid',):
diff --git a/coldfront/core/allocation/tasks.py b/coldfront/core/allocation/tasks.py
index e1a37fc..ba96800 100644
--- a/coldfront/core/allocation/tasks.py
+++ b/coldfront/core/allocation/tasks.py
@@ -29,7 +29,7 @@ EMAIL_ADMIN_LIST = import_from_settings('EMAIL_ADMIN_LIST')
 def update_statuses():
 
     expired_status_choice = AllocationStatusChoice.objects.get(
-        name='Expired')
+        name='Active (Needs Renewal)')
     allocations_to_expire = Allocation.objects.filter(
         status__name__in=['Active','Payment Pending','Payment Requested', 'Unpaid',], end_date__lt=datetime.datetime.now().date())
     for sub_obj in allocations_to_expire:
