diff --git a/coldfront/core/allocation/signals.py b/coldfront/core/allocation/signals.py
index 1c078f0..51277b1 100644
--- a/coldfront/core/allocation/signals.py
+++ b/coldfront/core/allocation/signals.py
@@ -14,3 +14,6 @@ allocation_remove_user = django.dispatch.Signal()
 
 allocation_change_approved = django.dispatch.Signal()
     #providing_args=["allocation_pk", "allocation_change_pk"]
+
+allocation_change_created = django.dispatch.Signal()
+    #providing_args=["allocation_pk", "allocation_change_pk"]
diff --git a/coldfront/core/allocation/views.py b/coldfront/core/allocation/views.py
index ec129fc..fa9b39c 100644
--- a/coldfront/core/allocation/views.py
+++ b/coldfront/core/allocation/views.py
@@ -53,6 +53,7 @@ from coldfront.core.allocation.signals import (allocation_new,
                                                allocation_activate_user,
                                                allocation_disable,
                                                allocation_remove_user,
+                                               allocation_change_created,
                                                allocation_change_approved,)
 from coldfront.core.allocation.utils import (generate_guauge_data_from_usage,
                                              get_user_resources)
@@ -1811,6 +1812,12 @@ class AllocationChangeView(LoginRequiredMixin, UserPassesTestMixin, FormView):
 
         messages.success(request, 'Allocation change request successfully submitted.')
 
+        allocation_change_created.send(
+            sender=self.__class__,
+            allocation_pk=allocation_obj.pk,
+            allocation_change_pk=allocation_change_request_obj.pk,)
+        
+
         send_allocation_admin_email(allocation_obj,
                                     'New Allocation Change Request',
                                     'email/new_allocation_change_request.txt',
